pkg/sfu/receiver.go

```diff
@@ -645,6 +645,14 @@ func (w *WebRTCReceiver) forwardRTP(layer int32) {
 			}
 		}
 
+		w.downTrackSpreader.Broadcast(func(dt TrackSender) {
```

Comment:
Move forward packet before track observer to fix a svc case that might generate additional pli:
1. publisher has active target bitmask indicates max layer s1t1
2. a key frame with max layer s2t2 arrived, observed by the streamtracker then it check sync for s2t2 in downtrack but failed as the key frame has not forwarded yet, send pli for sync
3. broadcast key frame to downtrack 
4. another keyframe generate by pli in step2 arrive again.

The change will swap 2 & 3 so downtrack will get key frame first and the tracker will not send pli as the `CheckSync` will return true.